### PR TITLE
FIX Update CMS fields now that they're being scaffolded

### PR DESCRIPTION
**Pull request by GuySartorelli**
Relies on changes to `SiteTree` in https://github.com/silverstripe/silverstripe-cms/pull/2983

## Issue
- https://github.com/silverstripe/silverstripe-cms/issues/2767

### CODE REVIEW

File: lang/en.yml
Comment (GuySartorelli) [inline lang/en.yml:1]: See https://github.com/silverstripe/silverstripe-cms/pull/2983#discussion_r1713062292 about translation strings.

File: src/IFramePage.php [hunk 1]
```diff
@@ -46,52 +47,40 @@ class IFramePage extends Page
...
+            );
+
+            $contentField = $fields->dataFieldByName('Content');
+            if ($contentField) {
+                $fields->removeByName('Content');
+                $contentField->setTitle(_t(__CLASS__ . '.db_Content', 'Content (appears above iframe)'));
+                $fields->addFieldToTab('Root.Main', $contentField, 'BottomContent');
+            }
+            $fields->dataFieldByName('BottomContent')?->addExtraClass('stacked');
+            $fields->dataFieldByName('AlternateContent')?->addExtraClass('stacked');
```
Comment (ChloeSartorelli) [inline src/IFramePage.php:78]: Intentionally made these wider, because it was silly how much space was being wasted

File: src/IFramePage.php [hunk 2]
```diff
@@ -46,52 +47,40 @@ class IFramePage extends Page
...
+                    ->setEmptyString('')
+                    ->setDescription(
+                        'Avoids mixed content warnings when iframe content is just available under a specific protocol'
+                    )
+            );
+
+            $contentField = $fields->dataFieldByName('Content');
+            if ($contentField) {
+                $fields->removeByName('Content');
+                $contentField->setTitle(_t(__CLASS__ . '.db_Content', 'Content (appears above iframe)'));
```
Comment (ChloeSartorelli) [inline src/IFramePage.php:74]: `fieldLabel()` only takes the class where the field is defined into account, not the class where it's being called from, so we can't just rely on `$this->fieldLabel('Content')` here like we normally would.

File: lang/en.yml
Comment (ChloeSartorelli) [inline lang/en.yml:1]: See https://github.com/silverstripe/silverstripe-cms/pull/2983#discussion_r1713062292 about translation strings.

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**